src/User_Command.php

```diff
@@ -418,6 +418,54 @@ public function create( $args, $assoc_args ) {
 	 * <user>...
 	 * : The user login, user email or user ID of the user(s) to update.
 	 *
+	 * --user_pass=<login>
```

Comment:
Oh, and this argument should use `<pass>` instead of `<login>`. Please make sure each placeholder matches the argument name.